demisto_sdk/commands/content_graph/objects/pack.py

```diff
@@ -113,11 +121,63 @@ class PackMetadata(BaseModel):
     preview_only: Optional[bool] = Field(None, alias='previewOnly')
 
 
-class Pack(BaseContent, PackMetadata):
+class Pack(BaseContent, PackMetadata, content_type=ContentType.PACK):
     path: Path
     contributors: Optional[List[str]] = None
     content_items: PackContentItems = Field(alias='contentItems', exclude=True)
     relationships: Relationships = Field(Relationships(), exclude=True)
 
+    def dump_metadata(self, path: Path) -> None:
+        metadata = self.dict(exclude={'path', 'node_id', 'content_type'})
```

Comment:
is it possible to explicitly include `contentItems` instead of the logic below?

Reply:
Tried it, but it didn't work. It includes only the `contentItems`